perception/autoware_tensorrt_yolox/src/tensorrt_yolox.cpp

```diff
@@ -156,16 +156,25 @@ namespace autoware::tensorrt_yolox
 TrtYoloX::TrtYoloX(
   const std::string & model_path, const std::string & precision, const int num_class,
   const float score_threshold, const float nms_threshold, tensorrt_common::BuildConfig build_config,
-  const bool use_gpu_preprocess, std::string calibration_image_list_path, const double norm_factor,
-  [[maybe_unused]] const std::string & cache_dir, const tensorrt_common::BatchConfig & batch_config,
-  const size_t max_workspace_size, const std::string & color_map_path)
+  const bool use_gpu_preprocess, const uint8_t gpu_id, std::string calibration_image_list_path,
+  const double norm_factor, [[maybe_unused]] const std::string & cache_dir,
+  const tensorrt_common::BatchConfig & batch_config, const size_t max_workspace_size,
+  const std::string & color_map_path)
+: gpu_id_(gpu_id)
 {
+  std::cout << "GPU " << gpu_id_ << " is selected for the inference!" << std::endl;
```

Comment:
```suggestion
  std::cout << "GPU " << std::to_string(gpu_id_) << " is selected for the inference!" << std::endl;
```
@ismetatabay -san,
This is the final request for modification. Because the current code output is like the following in my environment, I'd appreciate it if you fix it:
```
[autoware_tensorrt_yolox_node_exe-2] GPU  is selected for the inference!
```
Once you handle this, I'll approve this PR!

Reply:
Thank you @manato -san, it is done :+1: